testnet/stacks-signer/src/net.rs

```diff
@@ -0,0 +1,62 @@
+use crate::config::Config;
+use crate::signer;
+use serde::Serialize;
+use std::fmt::{Debug, Formatter};
+use ureq::Response;
+
+pub struct HttpNet {
+    _highwater_msg_idx: usize,
+    stacks_node_url: String,
+}
+pub trait Net {
+    fn listen(&self);
+    fn poll(&self) -> Result<Response, ureq::Error>;
+    fn next_message(&self) -> Message;
+    fn send_message(&self, _msg: Message);
+}
+
+#[derive(Debug)]
+pub struct Message {
+    pub r#type: signer::MessageTypes,
+}
+
+impl Debug for signer::MessageTypes {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Message Type: _")
```

Comment:
Shouldn't this print out the message instance, instead of `_`?

Reply:
the code was pushed over from trustmachines in an in-process state. i'll wrap up these loose ends and use a feature branch that merges into sbtc from here on.